lib/search/global/projects.js

```diff
@@ -11,6 +11,15 @@ module.exports = client => async (term = '', query = {}) => {
 
   params.body.query = { bool: {} };
 
+  params.body.highlight = {
+    fields: {
+      'title': { type: 'plain', pre_tags: '**', post_tags: '**' },
+      'establishment.name': { type: 'plain', pre_tags: '**', post_tags: '**' },
+      'licenceNumber': { type: 'plain', pre_tags: '**', post_tags: '**' },
+      'licenceHolder.lastName': { type: 'plain', pre_tags: '**', post_tags: '**' }
```

Comment:
It's probably easier to define these as:
```
fields: {
  '*': { type: 'plain', pre_tags: '**', post_tags: '**' }
}
```

Reply:
Wasn't aware you could do that, cheers.

Reply:
I'm not 100% sure, but I do `content.*` in the project search, so I'm assuming a plain `*` works too.